Makefile

```diff
@@ -168,6 +174,10 @@ publish-dockerhub: ## Mirror images on Docker Hub.
 	docker push docker.io/$(IMAGE_REPO)/keda:$(VERSION)
 	docker push docker.io/$(IMAGE_REPO)/keda-metrics-apiserver:$(VERSION)
 
+docker-delete: ## Delete images from Container Registry (default: ghcr.io).
+	docker image rm $(IMAGE_CONTROLLER)
```

Comment:
This would imho remove the images from the machine that it is running on, ie the GH worker. We should try to remove them from the remote registry instead. Not sure if that's possible, @tomkerkhove by chance do you know?

Reply:
You're right :(
I did it at the end and I (wrongly) assumed that it works.
I'll try to figure out if it's possible using ghcr, maybe there is any integration wirh github actions

Reply:
This has been solved using another GitHub Action